### PR TITLE
[bug](udf) fix udf return type of decimal check scale must is 9

### PR DESCRIPTION
**Pull request by zhangstar333**
# Proposed changes
`ERROR 1105 (HY000): errCode = 2, detailMessage = [INTERNAL_ERROR]UdfRuntimeException: UDF::evaluate() ran into a problem.
CAUSED BY: IllegalArgumentException: Scale of DECIMALV2 must be 9`
Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by carlvinhust2012** [comment]
What is the effect of this change?  Not report error？

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.56 seconds
 load time: 478 seconds
 storage size: 17170949839 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230208073831_clickbench_pr_91975.html

**Comment by zhangstar333** [comment]
> What is the effect of this change? Not report error？

remove the check, and set scale to 9 always

**Review by yiguolei — COMMENTED** [review]
Add some regression test for this?

File: fe/java-udf/src/main/java/org/apache/doris/udf/BaseExecutor.java [hunk 1]
```diff
@@ -386,5 +385,6 @@ protected boolean storeUdfResult(Object obj, long row, Class retClass) throws Ud
         }
     }
 
-    protected void updateOutputOffset(long offset) {}
+    protected void updateOutputOffset(long offset) {
+    }
```
Comment (HappenLee) [inline fe/java-udf/src/main/java/org/apache/doris/udf/BaseExecutor.java:389]: do not change unless code

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**